std_msgs/Header header
std_msgs/Bool indicator
std_msgs/Bool dropPrev